packages/synapse-react-client/src/components/AclEditor/AclEditor.test.tsx

```diff
@@ -174,4 +189,183 @@ describe('AclEditor', () => {
     expect(screen.queryByRole('combobox')).not.toBeInTheDocument()
     expect(screen.queryByRole('button')).not.toBeInTheDocument()
   })
+
+  it('allows customizing which entries are editable by passing a function to canEdit', async () => {
```

Comment:
In the Entity ACL editor, we must prevent users from editing the permission level of the public group ('Can view' only, unless it supports anonymous download access, in which case it is 'Can download' only).

Verify that passing a function to `canEdit` allows the parent component to apply such a restriction.